frontend/lib/identification/verification_workflow/verification_hasher.dart

```diff
@@ -0,0 +1,29 @@
+import 'dart:convert';
+import 'dart:typed_data';
+import 'package:crypto/crypto.dart';
+import 'package:ehrenamtskarte/proto/card.pb.dart';
+
+String hashVerifyCode(DynamicVerifyCode verifyCode) {
+  final hasher = Hmac(sha256, verifyCode.pepper);
+  final byteList = cardDetailsToBinary(verifyCode.info);
+  final result = hasher.convert(byteList);
+  return const Base64Encoder().convert(result.bytes);
+}
+
+List<int> cardDetailsToBinary(CardInfo cardInfo) {
```

Comment:
We could also use extension methods for this fn: https://dart.dev/guides/language/extension-methods

Reply:
Let's maybe do this in https://github.com/digitalfabrik/entitlementcard/issues/673